platform/ui/src/components/Select/Select.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import ReactSelect from 'react-select';
+
+import './Select.css';
+
+const Select = ({
+  autoFocus,
+  className,
+  classNamePrefix,
```

Comment:
This is not being used anywhere. Maybe it can be removed from props.

Reply:
Make sense, I am using it to force the classes to be the same so I can update the css using tailwind.